voucher/templates/enroll.html

```diff
@@ -10,18 +10,46 @@ <h1>Boeing employee enrollment page</h1>
             <h2>Confirm Course</h2>
             <hr/>
             <p>Please confirm this is the course you are trying to enroll in:</p>
+            {% if messages %}
+                {% for message in messages %}
+                <div{% if message.tags %} class="form-{{ message.tags }}"{% endif %}>{{ message }}</div>
+                {% endfor %}
+            {% endif %}
             <form method="post" enctype="multipart/form-data" class="voucher-form ">
                 {% csrf_token %}
-                <select name="coupon_version" id="coupon_version" classname="form-control">
-                    {% for choice in eligible_choices %}
-                        <option value="{{ choice.0 }}">{{ choice.1 }}</option>
-                    {% endfor %}
-                </select>
+                <div class="flex">
+                    <select name="coupon_version" id="coupon_version" classname="form-control">
+                        <option value="">Choose Course</option>
+                        {% for choice in eligible_choices %}
+                            <option value="{{ choice.0 }}">{{ choice.1 }}</option>
+                        {% endfor %}
+                    </select>
+                    <div id="voucher-error" class="form-error d-none">Please select an item in the list.</div>
```

Comment:
```suggestion
                    <div id="voucher-error" class="form-error d-none">Please select a course from the list. If your expected course isn't in the list, please contact <a href="https://xpro.zendesk.com/hc/en-us/requests/new" target="_blank" rel="noopener noreferrer">Customer Support</a></div>
```

Hopefully my HTML is correct...